packetery/src/Packetery/Plugin.php

```diff
@@ -158,7 +157,7 @@ public function __construct(
 	 * Method to register hooks
 	 */
 	public function run(): void {
-		$this->load_textdomains();
+		$this->loadTranslation();
```

Comment:
nemělo by to být spíše vyřešeno použitím : add_action( 'init', array( $this, 'loadTranslation' ) ) ?

Reply:
Nechci řešit problém, kdy překlad má být načtený ale načtený není. Když to jde hned načíst, tak je lepší to udělat tak.

Reply:
Jaký konkrétní problém by to přineslo.  Všechny ostatní moduly to takto používají.